Mathlib/Topology/Connected.lean

```diff
@@ -1380,7 +1379,7 @@ theorem isTotallyDisconnected_singleton {x} : IsTotallyDisconnected ({x} : Set 
 #align is_totally_disconnected_singleton isTotallyDisconnected_singleton
 
 /-- A space is totally disconnected if all of its connected components are singletons. -/
-class TotallyDisconnectedSpace (α : Type u) [TopologicalSpace α] : Prop where
+class TotallyDisconnectedSpace (α : Type*) [TopologicalSpace α] : Prop where
   /-- The universal set `Set.univ` in a totally disconnected space is totally disconnected. -/
```

Comment:
These changes save one character in the source in exchange for two more characters (`u_1` and `u`) in the docs and goal view; so I'm not sure they're worthwhile.